### DISCUSSION TITLE
How do I interpret/parse the date that contains this font?

### DISCUSSION THREAD

**Discussion by exo-cortex** [idleberg/playdate-arcade-fonts#4] [General]
Hi, I have a little project where I might be able to use some of these nice fonts. 
But I don't fully understand how they are saved. I see there is "data" in the files, but how does that work? Is that a certain file format that I just don't know about?

edit: I was trying to write "data" in the title. Cannot edit the title.

**Comment by gingerbeardman** [comment]
They are in the standard format for Playdate SDK. Are you developing for Playdate?

But the data in the file is base64 encoded PNG. But if all you need is the PNG there are probably easier ways to get that.

**Reply by exo-cortex** [reply]
Hi, 
no I'm not developing for playdate. I am trying to get text on a series of 8x8 LED displays. Ideally I'd have the letters as arrays of 8 bytes (so 8 rows, one byte per row, one bit per pixel). 
But i will probably find a way to convert them.

**Reply by gingerbeardman** [reply]
You can load these files into Bits'n'Picas font app, which may allow you to save in the source code format you want (or one very similar, or that can be more easily converted like BDF) https://github.com/kreativekorp/bitsnpicas

<img width="434" alt="Screen shot 2024-07-01 at 21 03 23" src="https://github.com/idleberg/playdate-arcade-fonts/assets/49612/b99e78b6-70ca-4ab9-989e-f99da4c6abd9">

**Reply by idleberg** [reply]
I've added an entry to the FAQ recommending Bits'n'Picas to convert these fonts. Thanks for sharing the link (again)!
